wpilibj/src/main/java/edu/wpi/first/wpilibj/RobotBase.java

```diff
@@ -347,7 +347,7 @@ public static boolean getBooleanProperty(String name, boolean defaultValue) {
       m_runMutex.unlock();
       if (!suppressExitWarning) {
         // startCompetition never returns unless exception occurs....
-        DriverStation.reportWarning("Robots should not quit, but yours did!", false);
+        DriverStation.reportWarning("An unexpected error has occurred. Please read the following stacktrace and verify your robot program", false);
```

Comment:
Should be "above" not "following" (see the reportError message a few lines down, which refers to "handled the exception above").

Reply:
Done, it should be good now.